packages/web-components/package.json

```
@@ -96,11 +98,16 @@
     "e2e:local": "node ./scripts/e2e.js --ui"
   },
   "devDependencies": {
+    "@custom-elements-manifest/analyzer": "^0.10.4",
     "@fluentui/scripts-api-extractor": "*",
     "@microsoft/fast-element": "2.0.0",
     "@tensile-perf/web-components": "~0.2.0",
     "@storybook/html": "7.6.20",
     "@storybook/html-webpack5": "7.6.20",
+    "@wc-toolkit/cem-validator": "^1.0.2",
+    "@wc-toolkit/cem-inheritance": "^1.0.4",
+    "@wc-toolkit/module-path-resolver": "^1.0.0",
+    "@wc-toolkit/type-parser": "^1.0.3",
```

Ditto here, and then you'll just need to run `yarn` again to update the lockfile and the errors should go away
```suggestion
    "@wc-toolkit/cem-validator": "1.0.2",
    "@wc-toolkit/cem-inheritance": "1.0.4",
    "@wc-toolkit/module-path-resolver": "1.0.0",
    "@wc-toolkit/type-parser": "1.0.3",
```